### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dA505819**
Restructured content.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,11 +96,17 @@ You can open these notebooks directly in [Google Colab](./notebooks/google-colab
...
+# License
+
+Refer [licensing](LICENSE.md) (and warranty) policy.
+
 # Contributing
 
-Contributions are very welcome, please share back with the wider community (and get credited for it)!
+Contributions are Welcome!
+
+Please ave a look at the [CONTRIBUTING](CONTRIBUTING.md) guidelines.
```
Comment (neomatrix369) [inline README.md:107]: > Please ave a
Please have a

File: README.md [hunk 2]
```diff
@@ -96,11 +96,17 @@ You can open these notebooks directly in [Google Colab](./notebooks/google-colab
...
 
-Please have a look at the [CONTRIBUTING](CONTRIBUTING.md) guidelines, also have a read about our [licensing](LICENSE.md) (and warranty) policy.
+Please share it with the wider community (and get credited for it)!
```
Comment (neomatrix369) [inline README.md:109]: 👍

File: README.md [hunk 3]
```diff
@@ -72,7 +72,7 @@ See [Notebooks](#Notebooks) section for further illustrations.
 
 ## Notebooks
 
-_Once the library is successfully installed, please RESTART your Jupyter kernels or Google Colab runtimes for the changes to take effect._
+_After succesful installtion of library, RESTART Jupyter kernels or Google Colab runtimes for the changes to take effect._
```
Comment (neomatrix369) [inline README.md:75]: > After succesful installtion of library
After successful installation of library

File: README.md [hunk 4]
```diff
@@ -2,16 +2,17 @@
 
 A simple NLP library allows profiling datasets with one or more text columns. 
 
-When given a dataset and a column name containing text data, NLP Profiler will return either high-level insights or low-level/granular statistical information about the text in that column. 
+NLP Profiler returns either high-level insights or low-level/granular statistical information about the text when given a dataset and a column name containing text data,  in that column.
```
Comment (neomatrix369) [inline README.md:5]: > containing text data,  in that column.
containing text data, in that column.

**Review by neomatrix369 — CHANGES_REQUESTED** [review]
Thank you @dA505819 
Looks good. Nice changes.

Do you mind also adding a ToC at the top of the README, I should have done it while working it.

I like the way this repo has it's ToC: https://github.com/jbangdev/jbang but we can keep it simple for now and expand on it later

File: README.md (hunk 4 above)
Comment (dA505819) [reply README.md]: Resolved

File: README.md (hunk 2 above)
Comment (dA505819) [reply README.md:109]: Corrected file.

**Review by dA505819 — COMMENTED** [review]
Corrected file

File: README.md (hunk 3 above)
Comment (neomatrix369) [reply README.md]: > After successful installation of library
After successful installation of the library -- my bad I didn't notice @dA505819

**Comment by neomatrix369** [comment]
@dA505819 one last change, you can do the ToC via another PR if you like

**Comment by neomatrix369** [comment]
Thank you very much @dA505819

**Comment by dA505819** [comment]
@neomatrix369 Sure!:)

File: README.md (hunk 3 above)
Comment (dA505819) [reply README.md]: Done

**Comment by dA505819** [comment]
@neomatrix369 I will add TOC in next PR.

**Review by neomatrix369 — APPROVED** [review]
Thanks @dA505819

**Comment by dA505819** [comment]
@neomatrix369 You're Welcome Mani!

**Pull request merged**